WORKSPACE

```diff
@@ -17,6 +17,14 @@ workspace(name = "rules_python")
 # Everything below this line is used only for developing rules_python. Users
 # should not copy it to their WORKSPACE.
 
+# Necessary so that Bazel 9 recognizes this as rules_python and doesn't try
+# to load the version Bazel itself uses by default.
+# buildifier: disable=duplicated-name
```

Comment:
This definitely looks like a bug within `bazel`. It could check the `workspace name` and do it itself, but I am OK with adding this into our repo to get things working.

Reply:
I filed https://github.com/bazelbuild/rules_python/issues/2469 to bazel, for at least posterity.